paypro-gateways-woocommerce/includes/paypro/wc/gateway.php

```diff
@@ -1,18 +1,15 @@
 <?php if(!defined('ABSPATH')) exit; // Exit if accessed directly
 
-abstract class PayPro_WC_Gateway_Abstract extends WC_Payment_Gateway
+class PayPro_WC_Gateway extends WC_Payment_Gateway
```

Comment:
Should we not keep this `abstract`? We do not want this to be instantiated by itself.

Reply:
Could you explain a bit more, i don't get it